src/Sql/dbo/Stored Procedures/SsoConfig_ReadByIdentifier.sql

```diff
@@ -5,9 +5,12 @@ BEGIN
     SET NOCOUNT ON
 
     SELECT TOP 1
-        *
+        SSO.*,
+        O.[Identifier]
```

Comment:
You're going to have to create a special model for this sproc since it returns one additional field, which is ok, but wanted to make sure we needed it.

Reply:
That's a good point, I assumed we would need it when it came to pulling configurations in bulk for all organizations + caching them vs. fetch on-demand, but we've not necessarily identified that as a pattern so for now to keep things simple we can remove it @mportune-bw . Sorry for the thrash on this one.

Reply:
No worries!